src/app/calculator/page.tsx

```diff
@@ -0,0 +1,30 @@
+"use client";
```

Comment:
make sure this isn't merged to v3 as it's just for testing that the radio group works. 💡